_data/chains/eip155-31612.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Mezo",
+  "chain": "Mezo",
+  "rpc": [
+    "https://rpc_evm-mezo.imperator.co",
+    "wss://ws_evm-mezo.imperator.co",
+    "https://jsonrpc-mezo.boar.network",
+    "wss://jsonrpcws-mezo.boar.network"
+  ],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "Bitcoin",
+    "symbol": "BTC",
+    "decimals": 18
+  },
+  "features": [{ "name": "EIP155" }, { "name": "EIP1559" }],
+  "infoURL": "https://mezo.org/",
+  "shortName": "mezo",
```

Comment:
Shortname also duplicated

Reply:
Fixed in 7d976e822f56be9d341899cda7873e6ddc0367ce. I realized your comment was about it at the same moment I pushed the commit. 🙂 It feels like the `shortName` has to be unique, but `chain` doesn't have to, right? At least this is what I get looking at other examples, like 80094 and 80069.

Anyway, probably better to keep them all different to avoid any confusion.